### DISCUSSION TITLE
L-BFGS freezes with 'tensorflow'

### DISCUSSION THREAD

**Discussion by pescap** [lululxvi/deepxde#700] [Q&A] [answered]
Hi, 
https://github.com/lululxvi/deepxde/blob/4cd2178dfc70e06c629c858870a5f64f622982b6/examples/pinn_forward/Poisson_Lshape.py#L25 freezes when using `tensorflow` as a backend, in CPU and GPU modes.

Example with 5000 ADAM epochs: 
```
'train' took 58.066181 s

Compiling model...
'compile' took 0.019574 s

Training model...

Step      Train loss              Test loss               Test metric
5000      [6.19e-04, 5.82e-04]    [6.10e-04, 5.82e-04]    [] 
```
and then, nothing.

**Comment by lululxvi** [comment, marked as answer]
It is still running. Just wait.

**Reply by ShaikhaTheGreen** [reply]
I have another question related to L-BFGS. In some of my runs it only performs one step of iterations (1000 epochs) and then the training concludes. Is this normal?

**Reply by lululxvi** [reply]
See https://deepxde.readthedocs.io/en/latest/modules/deepxde.optimizers.html#deepxde.optimizers.config.set_LBFGS_options
